templates/settings_page.php

```diff
@@ -44,55 +45,67 @@
             <div class="pmb-reveal-details">
                 <table class="form-table">
                     <tbody>
-                    <tr>
-                        <th scope="row">
-                            <label for="pmb-place-above"><?php esc_html_e('Place Buttons', 'print-my-blog');?></label>
-                        </th>
-                        <td>
-                            <select name="pmb_place_above" id="pmb-place-above">
-                                <option value="1" <?php echo $settings->showButtonsAbove() ? 'selected="selected"' : '';?>><?php esc_html_e('Above Content', 'print-my-blog');?></option>
-                                <option value="0" <?php echo ! $settings->showButtonsAbove() ? 'selected="selected"' : '';?>><?php esc_html_e('Below Content', 'print-my-blog');?></option>
-                            </select>
-                        </td>
-                    </tr>
+                        <tr>
+                            <th scope="row">
+                                <label for="pmb-place-above"><?php esc_html_e('Place Buttons', 'print-my-blog'); ?></label>
+                            </th>
+                            <td>
+                                <select name="pmb_place_above" id="pmb-place-above">
+                                    <option value="1" <?php echo $settings->showButtonsAbove() ? 'selected="selected"' : ''; ?>><?php esc_html_e('Above Content', 'print-my-blog'); ?></option>
+                                    <option value="0" <?php echo ! $settings->showButtonsAbove() ? 'selected="selected"' : ''; ?>><?php esc_html_e('Below Content', 'print-my-blog'); ?></option>
+                                </select>
+                            </td>
+                        </tr>
                     </tbody>
                 </table>
-                <h2><?php esc_html_e('Print Formats', 'print-my-blog');?></h2>
-                <?php foreach($settings->formats() as $slug => $format){ ?>
+                <h2><?php esc_html_e('Preview Settings', 'print-my-blog'); ?></h2>
```

Comment:
I wouldn't bother saying "Preview Settings", especially when there's just one setting relating to this
```suggestion
                <h2><?php esc_html_e('Preview Settings', 'print-my-blog'); ?></h2>
```

Reply:
Sorry, I had VS Code set to auto-format on save. I turned it off while working on this, but I must have missed it for this file. 

The default setting is in FrontendPrintSettings.php on Line 105. I had set it to "false" originally, but looks like I left it on "true" during testing. I will fix that. Thank you for catching that.